.github/workflows/tests.yaml

```diff
@@ -537,10 +537,15 @@ jobs:
                       --target linux-x64-light-data-model-no-unique-id-${BUILD_VARIANT}-clang \
                       --target linux-x64-terms-and-conditions \
                       --target linux-x64-python-bindings \
+                      --target linux-x64-jf-control-app \
+                      --target linux-x64-jf-admin-app \
```

Comment:
these should use ${BUILD_VARIANT} like the other apps. We want to ensure consistent settings everywhere.

Reply:
maybe tsan-clang like the other test apps too.